test/sharness/t0027-rotate.sh

```diff
@@ -4,13 +4,23 @@ test_description="Test rotate command"
 
 . lib/test-lib.sh
 
-# $1 must be one of 'rsa', 'ed25519' or '' (for default key algorithm).
 test_rotate() {
-        TEST_ALG=$1
+        FROM_ALG=$1
+        TO_ALG=$2
```

Comment:
This is fine, although I think we can simplify it so that we:
1. first test basic rotation on default, RSA, Ed25519 using the test_rotate function which indicates that rotation basically works
2. test rotating Ed25519 -> RSA -> Ed25519 which indicates that we can rotate to/from each key type. We can even use `test_expect_success` for these steps to make them easier to debug/notice.